bitcoin/test/miniscript_tests.cpp

```diff
@@ -605,6 +605,13 @@ BOOST_AUTO_TEST_CASE(fixed_tests)
     std::vector<unsigned char> nonminpush = ParseHex("0000210232780000feff00ffffffffffff21ff005f00ae21ae00000000060602060406564c2102320000060900fe00005f00ae21ae00100000060606060606000000000000000000000000000000000000000000000000000000000000000000");
     const CScript nonminpush_script(nonminpush.begin(), nonminpush.end());
     BOOST_CHECK(miniscript::FromScript(nonminpush_script, CONVERTER) == nullptr);
+    // This Script does not decode to a valid Miniscript, but if it didn't contain a
+    // non-minimal VERIFY sequence (OP_EQUAL OP_VERIFY here) it would at least get through
+    // the DecomposeScript() stage.
+    std::vector<unsigned char> nonminverify = ParseHex("63006321027d86781855db1b17d7ce316d816076eba7eac3bb91447d460438778f8db0e5bb006b006c930000000000016c9301000000000000006b63006751686c936b630067630092670068686c936b006c936b63006751686c935387690000009a68");
```

Comment:
Perhaps this: "03a0434d9e47f3c86235477c7b1ae6ae5d3442d49b1943c2b752a68e2a47e247c7ac6951" (`<key> CHECKSIG VERIFY 1`)?

Reply:
Yeah, this is much better :sweat_smile: 

Tested it was accepted before the patch, not after. Amended the commit